nx.json

```diff
@@ -141,14 +141,19 @@
       "options": {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33028&commitId=2e9d98341ccdb6042cad013cd7181cd148f97430&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.Top right edge.chromium.png|1594| Changed|
|Callout.Bottom right edge.chromium.png|1596| Changed|
|Callout.Left center.chromium.png|2616| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>